tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "root": true,
   "extends": "@fullstacksjs/tsconfig/react",
-  "include": ["src", "@types"],
+  "include": ["src", "@types", "vite.config.ts"],
```

Comment:
Why should we add vite.config here?

Reply:
I don't remember why I added it
but if it works without it then we can remove it